drivers/encx24j600/encx24j600.c

```diff
@@ -360,7 +360,7 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
     /* hdr.frame_len given by device contains 4 bytes checksum */
     size_t payload_len = hdr.frame_len - 4;
 
-    if (buf) {
+    if (buf && (len >= payload_len)) {
```

Comment:
@kaspar030 so basically if the procedure is followed everywhere, the `len` parameter would be obsolete anyway - or why is it there?